src/popup.tsx

```diff
@@ -1,13 +1,19 @@
 import React, { useEffect, useState } from "react";
 import { createRoot } from "react-dom/client";
-import { FormattedMessage } from "react-intl";
-import { Intl } from "./providers/Intl";
+import { LocationDropdown, LocationType } from './components/LocationDropdown'
+import { LOCATIONS } from "./constants/locations";
+import { calculateCarbon } from "./helpers/calculateCarbon";
+
 
 export const Popup = () => {
```

Comment:
I think we can export all of the logic out to use the 1 hook per screen rule ;) (we will talk about this in coding standards meeting